### DISCUSSION TITLE
玩具店门口的怪人

### DISCUSSION THREAD

**Discussion by FeiLiFL** [share121/inter-knot#5821] [公告]
![怪人](https://github.com/user-attachments/assets/f3ab81c7-4335-419f-b5fc-cf5c421a8813)
今天早上在404live门口看见这个绿油油的身影坐在那里，下午我去的时候还在！她是在那里坐了一整天吗？（惊诧）应该不是在等人吧？不然对方岂不是放了她一天鸽子？

**Comment by Yielin** [comment]
哦！是安比啊，平常我都在光映的汉堡店见到她的，我就说今天怎么没见到她呢。

**Reply by FeiLiFL** [reply]
这位朋友，听上去你好像和她很熟？有联系方式吗？

**Reply by Yielin** [reply]
不能说很熟啦，就是一同吃饭的朋友，偶尔聊聊天，如果要找她的话建议去狡兔屋问问。她好像在那里工作。

**Reply by FeiLiFL** [reply]
狡兔屋的代理人。。。他们老大还欠我两次委托的钱呢。。。

**Reply by Mai-ther** [reply]
> 狡兔屋的代理人。。。他们老大还欠我两次委托的钱呢。。。

我也是……上次我去问债务的事，还被一个眼角皱纹不少的小哥警告了

**Comment by feather000A** [comment]
到处都是语C佬
